core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -352,7 +356,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
   }
 
   @Test
-  def testConsumeWithNoTopicAccess() {
+  def testConsumeWithoutTopicDescribeAccess() {
```

Comment:
The expectation of this test case is not too clear. It looks like the exception that we raise is actually being thrown from the offset fetch request, and it seems like it may be a separate bug that we don't handle the UNKNOWN_TOPIC_OR_PARTITION error code in the offset fetch response handler. Maybe we should fix that? Ultimately, until we have timeouts for some of the consumer's blocking calls, we may not have a great way to test this case.